components/drawer/__tests__/__snapshots__/Drawer.test.js.snap

```diff
@@ -40,7 +39,6 @@ exports[`Drawer destroyOnClose is true 1`] = `
     />
     <div
       class="ant-drawer-content-wrapper"
-      style="transform: translateX(100%);"
```

Comment:
Just rebase again to remove these changes

Reply:
sorry @afc163 how do I do this?

Reply:
rebase feature-3.8.0

Reply:
I think now it has.